src/components/TodoItem.tsx

```diff
@@ -11,14 +13,22 @@ type TodoItemProps = {
 };
 
 const TodoItem = (props: TodoItemProps) => {
-  const { text, creationDate, expirationDate } = props.item;
+  const { text, creationDate, expirationDate, id, done } = props.item;
+  const dispatch = useDispatch();
+
+  const togglehandler = () => {
+    dispatch(todosActions.toggleTodo(id));
+  };
+
+  const crossed = `${done ? classes.crossed : ''}`;
 
   return (
     <li className={classes.item}>
-      <div className={classes.text}>{text}</div>
+      <input type="checkbox" checked={done} onChange={togglehandler} />
+      <div className={`${classes.text} ${crossed}`}>{text}</div>

```

Comment:
you can use destructuring for classes also. To not have classes.*** all the time